### PR TITLE
Remove MSAL Angular v1, v2, and AngularJS samples from dev

### PR DESCRIPTION
**Pull request by jo-arroyo**
Older MSAL Angular samples will continue to live on the LTS branch [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/msal-lts/samples).

References to samples have been updated to our angular15-sample-app where possible. Note that the angular15 and angular 16 samples do not demonstrate some older or lesser used features (e.g. hash routing), so examples for those situations may still be in the older v2 samples.

### CODE REVIEW

File: lib/msal-angular/docs/FAQ.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ For those using the `HashLocationStrategy`, we recommend:
 - Making sure the `MsalRedirectComponent` is bootstrapped
 - Optionally: adding `MsalGuard` to all your routes if you want all your routes protected
 
-See our [Angular 11 sample](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-angular-v2-samples/angular11-sample-app/src/app/app-routing.module.ts) for examples of how to protect routes with `MsalGuard`.
+See our [Angular 11 sample](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/msal-lts/samples/msal-angular-v2-samples/angular11-sample-app/src/app/app-routing.module.ts) for examples of how to protect routes with `MsalGuard`.
```
Comment (tnorling) [inline lib/msal-angular/docs/FAQ.md]: Are we going to replace these with newer samples for v3?

File: lib/msal-angular/docs/v2-docs/events.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ ngOnInit(): void {
 }
 ```
 
-For the full example of using events, please see our sample [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/d46c4455243bd51767f669a13fbb717d786c6716/samples/msal-angular-v2-samples/angular11-sample-app/src/app/home/home.component.ts#L17).
+For the full example of using events, please see our sample [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-angular-v3-samples/angular15-sample-app/src/app/home/home.component.ts).
```
Comment (tnorling) [inline lib/msal-angular/docs/v2-docs/events.md:59]: This is in the v2 docs folder but pointing to a v3 sample?

File: lib/msal-angular/docs/v2-docs/events.md [hunk 3]
```diff
@@ -101,7 +101,7 @@ export class AppComponent implements OnInit, OnDestroy {
 }
 ```
 
-An example of error handling can also be found on our [MSAL Angular v2 B2C Sample App](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/4d79b8ebacd7e4d9acf80fd69d602346dee6bf3c/samples/msal-angular-v2-samples/angular11-b2c-sample/src/app/app.component.ts#L68).
+An example of error handling can also be found on our [MSAL Angular v2 B2C Sample App](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/msal-lts/samples/msal-angular-v2-samples/angular11-b2c-sample/src/app/app.component.ts#L68).
```
Comment (tnorling) [inline lib/msal-angular/docs/v2-docs/events.md]: And then there's a v2 sample here in the same doc...we should be consistent.

File: lib/msal-angular/docs/FAQ.md (hunk 1 above)
Comment (jo-arroyo) [reply lib/msal-angular/docs/FAQ.md]: I have replaced as many with newer samples as possible. As we get more samples for v3, I may be able to phase out more older samples in docs.

File: lib/msal-angular/docs/v2-docs/events.md (hunk 2 above)
Comment (jo-arroyo) [reply lib/msal-angular/docs/v2-docs/events.md:59]: Good point. I ended up moving all docs out of the v2 folder instead.

**Review by peterzenz — APPROVED** [review]

File: change/@azure-msal-angular-a3675107-cba6-4d27-8953-d8503c06a105.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```
Comment (tnorling) [inline change/@azure-msal-angular-a3675107-cba6-4d27-8953-d8503c06a105.json]:
```suggestion
  "type": "none",
```

**Review by tnorling — APPROVED** [review]

**Pull request merged**